### PR TITLE
implemented insert and update fully. resolve #15

### LINKED ISSUE #15: Implement INSERT, UPDATE, DELETE basics
Those should now be easily doable. Create implementation and necessary tests.

### PR DESCRIPTION
**Pull request by romaninsh**

### CODE REVIEW

File: src/Query.php [hunk 1]
```diff
@@ -519,6 +540,92 @@ public function _render_having()
     }
     // }}}
 
+    // {{{ Set field implementation
+    /**
+     * Sets field value for INSERT or UPDATE statements.
+     *
+     * @param string $field Name of the field
```
Comment (DarkSide666) [inline src/Query.php]: $field can also be array

Comment (romaninsh) [inline src/Query.php]: agree. test case missing also.

**Comment by romaninsh** [comment]
I've restored and merged develop, although it was a bit messy, the "files changed" looks clean now.

I have also realised that I don't have documetation for this PR (or the previous), so i need to add that before merging.

**Comment by romaninsh** [comment]
Added documentation as promised. should be OK to merge, but don't forget to review.

**Pull request merged**